pvlib/modelchain.py

```diff
@@ -328,26 +328,26 @@ class ModelChain:
 
     dc_model: None, str, or function, default None
         If None, the model will be inferred from the contents of
-        system.module_parameters. Valid strings are 'sapm',
+        system.arrays[i].module_parameters. Valid strings are 'sapm',
```

Comment:
```suggestion
        If None, the model will be inferred from the parameters that
        are common to all of system.arrays[i].module_parameters. 
        Valid strings are 'sapm',
```

May help a user debug an odd case where the intersection is missing some required parameter, because one of `system.arrays[i].module_parameters` is incomplete.

If acceptable, same comment applies to the description of `ac_model`, `aoi_model`, `spectral_model`